styleSheet.css

```diff
@@ -236,6 +236,58 @@ ul {
   width: 100%;
 }
 
+#awardsInterest {
+  background-color: var(--dark__primary);
+  font-size: 1.2em;
+  height: calc(60vh + 20vw);
+  display: inline-flex;
```

Comment:
order elements alphabetically

Reply:
Set, Fixed, there is always a few that I miss